lib/mixpanel-ruby/consumer.rb

```diff
@@ -225,9 +225,16 @@ def flush
     private
 
     def flush_type(type)
-      @buffers[type].each_slice(@max_length) do |chunk|
-        data = chunk.map {|message| JSON.load(message)['data'] }
-        @sink.call(type, {'data' => data}.to_json)
+      sent_messages = 0
+      begin
+        @buffers[type].each_slice(@max_length) do |chunk|
+          data = chunk.map {|message| JSON.load(message)['data'] }
+          @sink.call(type, {'data' => data}.to_json)
+          sent_messages += @max_length
```

Comment:
Shouldn't this be `chunk.length`?

Reply:
in practice it doesn't make a difference, but good catch